daemon/src/main/java/org/apache/maven/project/SnapshotModelCacheFactory.java

```diff
@@ -45,6 +45,9 @@ public SnapshotModelCacheFactory(DefaultModelCacheFactory factory) {
 
     @Override
     public ModelCache createCache(RepositorySystemSession session) {
-        return new SnapshotModelCache(globalCache, factory.createCache(session));
+        boolean enableModelCache = Boolean.parseBoolean(System.getProperty("mvnd.modelCache", "true"));
```

Comment:
Could you please add the new property to `org.mvndaemon.mvnd.common.Environment` and add a bit of JavaDoc describing what it is doing?

Naming: Looking at other boolean properties, we have in `org.mvndaemon.mvnd.common.Environment`, perhaps the `mvnd.noModelCache` style would suit best here. I am not especially happy about negating names, but we already have a couple of those, and they are easier to use (given the default is with model cache enabled) from command line: `-Dmvnd.noModelCache` vs. `-Dmvnd.modelCache=false`. It is less typing. What do others think?

Reply:
Sure, I'll do that